<launch>
  <arg name="host" />
  <arg name="port" />
  <arg name="ssl_certfile" />
  <arg name="ssl_keyfile" />
  <node name="dialogflow_webhook_ros_server" pkg="dialogflow_webhook_ros" type="webhook_server.py">
    <rosparam subst_value="true">
      host: $(arg host)
      port: $(arg port)
      ssl_certfile: $(arg ssl_certfile)
      ssl_keyfile: $(arg ssl_keyfile)
    </rosparam>
  </node>
</launch>
